### ISSUE TITLE
Add to Android TV store

### ISSUE THREAD

**Issue by biskyt** [sds100/KeyMapper#488]
Hi, I am using your app on the new "Chromecast with Google TV" to remap the Netflix button on the remote as Play/Pause 

It works great.

Is there any chance you could make the app show up in the Android TV Play store, so other people can find it easier and not have to go through the extra steps of side-loading. It would also mean that auto-update would work on TV devices.

Thank you

**Comment by sds100** [comment]
Yeah, I'm planning on optimising the app for Android TV. Are there any usability issues? E.g are there any difficult to press buttons in the UI?

**Comment by UraniumDonut** [comment]
Delete button inaccessible from remote (for general action, and right side of editing), the rest works perfectly

**Comment by scatterthought** [comment]
Great app! I just sideloaded it onto my Chromecast with Google TV, and look forward to it being natively available in the Android TV store (hopefully with the volume-button issues solved).

From a usability standpoint, I could personally benefit from more contrast on selected items. Due to my red-green colorblindness, I have a hard time seeing when the red "Record Trigger" and blue "Add Action" buttons are selected, since they just become a slightly lighter shade.

Thanks for your work!

**Comment by sds100** [comment]
@scatterthought I'm glad you like the app! 😃 Is #453 the volume issue that you mentioned? I'm happy to add a setting that changes the colours. Which colours would be better? Does turning on the dark theme in the settings help?

Have a good day!

**Comment by scatterthought** [comment]
Yes, #453 is the volume issue I was referring to. The power button also doesn't work. I wonder if It's related to Android TV having options to remap these buttons its own settings?

I'll try the dark mode. I didn't see that, but I wasn't looking for it. It would definitely help if both buttons got really dark shades or red blue when selected. I'm happy to test combinations.

**Comment by sds100** [comment]
@scatterthought Android doesn't allow apps to remap the power button for a good reason. 😄 I've created an issue for the new theme at #601

**Comment by scatterthought** [comment]
Makes sense to me that the power button isn't available.

Funny thing about the dark mode: I couldn't see it in the settings at first, because it's the very first option at the top of the screen. So it looked more like a title bar, and I just started scrolling down to look for it. Unfortunately, it didn't help since the red stays the same and the blue is swapped out for green, which is even harder for me.

I'll comment more in #601. Thanks!

**Comment by Tahvok** [comment]
@sds100 don't want to bother much, but it's been some time. Can you please make it available on play store for google tv?
